source/apiVolontaria/volunteer/models.py

```diff
@@ -188,20 +188,25 @@ def set_permission_cell_manager(user, add=True):
     permission_delete_event = Permission.objects \
         .get(name='Can delete event')
 
+    permission_view_user = Permission.objects \
+        .get(name='Can view user')
+
```

Comment:
Is this permission exist ? In the DB i only see `add`, `change` and `delete`

Reply:
Yes, I have checked, here what it look like in the admin : 
<img width="401" alt="screen shot 2018-11-22 at 9 57 41 am" src="https://user-images.githubusercontent.com/22125499/48910143-2cfd1780-ee3d-11e8-8f46-24f4fd6a5be5.png">